asmcomp/power/arch.ml

```diff
@@ -33,6 +33,9 @@ type specific_operation =
   | Ialloc_far of                       (* allocation in large functions *)
       { bytes : int; dbginfo : Debuginfo.alloc_dbginfo }
   | Ipoll_far of { return_label : cmm_label option }
+                                        (* poll point in large functions *)
+  | Icheckbound_far                     (* bounds check in large functions *)
+  | Icheckbound_imm_far of int          (* bounds check in large functions *)
```

Comment:
Not suggesting a change in this PR, but observing that `arm64/arch.ml` calls these operations

```
| Ifar_alloc
| Ifar_poll
| Ifar_intop_checkbound
| Ifar_intop_imm_checkbound
```
It will be useful to standardize the names of these operations.

Reply:
I didn't write the ARM64 branch relaxation code, so I cannot comment on the choice of names that was made there.  Let me just point out that the relaxation code was first developed for POWER, with the name `Ialloc_far`, before it was applied to ARM64, with the `Ifar_*` names.  Then, poll points were added, with `Ipoll_far` name in POWER.  So, I'm just being consistent with the earlier POWER names.

Reply:
Not suggesting that we standardize to the ARM64 names. I prefer the `*_far` scheme used by POWER.